src/screens/App/screens/Detail/components/BookDetail/components/RentButton/index.js

```diff
@@ -9,7 +9,7 @@ const RentButton = ({userRent, enableWishList, onAddToWishlist, onRentBook}) =>
   if (userRent) {
     return (
       <div className='rent-button-container'>
-        <span>{`${RETURN_DATE} ${moment(userRent.to, 'YYYY-mm-DD').format('DD/mm/YYYY')}`}</span>
+        <span>{`${RETURN_DATE} ${formattedDate(userRent.to, 'YYYY-mm-DD')}`}</span>
```

Comment:
i mind, `formattedDate(userRent.to)`
and that `formattedDate` just use the format `YYYY-mm-DD`.

Reply:
but its ok too